examples/kubernetes/encryption_in_transit/specs/pv.yaml

```diff
@@ -10,8 +10,8 @@ spec:
     - ReadWriteOnce
   persistentVolumeReclaimPolicy: Retain
   storageClassName: efs-sc
-  mountOptions:
-    - tls
   csi:
     driver: efs.csi.aws.com
     volumeHandle: fs-4af69aab
+    volumeAttributes:
+      encryptInTransit: "true"
```

Comment:
Emm, if this is enabled by default, do we still need to set it explicitly in example? 

I think examples should provide a recommended usage pattern for users and I believe we encourage users to come with the mind set that encryptInTransit is enabled without them doing anything?

Reply:
IDK, depends how we wish to differentiate this example from the static_provisioning example. If we do not show how to set encryptInTransit then this example is pointless. I think it's fine just to have it in the pv yaml so users can easily toggle it by doing s/true/false/ & vice versa.

Reply:
I think it might make sense to set `encryptInTransit` to `false` in the static_provisioning example. Will send a PR after this is merged.